pgcli/packages/pgspecial.py

```diff
@@ -50,6 +50,68 @@ def list_schemas(cur, pattern, verbose):
         headers = [x[0] for x in cur.description]
         return [(cur, headers, cur.statusmessage)]
 
+def list_tables_or_views(cur, pattern, verbose, relkinds):
+    """
+        Returns (rows, header, status)
+
+        This method is used by list_tables and list_views.
+
+        relkinds is a list of strings to filter pg_class.relkind
+
+    """
+    schema_pattern, table_pattern = sql_name_pattern(pattern)
+
+    verbose_columns = ('''
```

Comment:
Minor nitpick: 

Would it be better to use: 

```
if verbose: 
    verbose_columns = ''' blah blah'''
else:
    verbose_columns = ''
```

I think it makes the code easier to follow. I only say this because I glossed over the if condition at the end of the long sql and I had to backtrack to see where you handled the non-versbose case.